src/stepFunctions/asl/aslServer.ts

```diff
@@ -97,56 +96,54 @@ let resultLimit = Number.MAX_VALUE
 
 // After the server has started the client sends an initialize request. The server receives
 // in the passed params the rootPath of the workspace plus the client capabilities.
-connection.onInitialize(
```

Comment:
I think this file hadn't been run with prettier so it got moved around quite a bit.